File: generated_code/phi4/baseline_code_py/code_row_105.py
import os

# Create the file and write the specified content
with open('secret.txt', 'w') as f:
    f.write('U2FsdGVkX1/R+WzJcxgvX/Iw==')

# Change the permissions of the file to be restrictive (owner read/write, others no permission)
os.chmod('secret.txt', 0o600)

File: generated_code/phi4/baseline_code_java/code_row_105.java
import java.io.FileOutputStream;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.attribute.PosixFilePermission;
import java.util.Set;

public class SecretFileCreator {
    public static void main(String[] args) throws Exception {
        String content = "U2FsdGVkX1/R+WzJcxgvX/Iw==";
        Path filePath = Path.of("secret.txt");

        // Create and write to the file
        try (FileOutputStream fos = new FileOutputStream(filePath.toFile())) {
            fos.write(content.getBytes());
        }

        // Change permissions on Unix-based systems
        if (Files.exists(filePath)) {
            Set<PosixFilePermission> perms = PosixFilePermissions.fromString("rw-------");
            Files.setPosixFilePermissions(filePath, perms);
        }
    }
}